### DISCUSSION TITLE
Pre-season temporary fix

### DISCUSSION THREAD

**Discussion by riffnshred** [riffnshred/nhl-led-scoreboard#315] [Announcements]
I had just push a temporary fix on the Beta branch for the pre-season games not showing. The current season is hardcoded and I removed the off-season loop in the renderer/main.py. I will have to re-write the status to handle the pre-season when the API is stuck in the previous season once the season started.

**Comment by xTrackstar04x** [comment]
Hey Riff. Just to let you know during intermission of a live game, if you have it cycling through game, the board crashes after the last game

I'm assuming it will be automatically be resolved once the season starts, but just wanted to give you the heads up

Version: Beta that was released yesterday (10/6)
